### DISCUSSION TITLE
Sono saltati gli autori?

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#23] [Domande] [answered]
Ciao @aborruso, per caso sono saltati gli autori o è "normale" in fase di transizione? https://opendatasicilia.it/author/opendat1/

![image](https://user-images.githubusercontent.com/77018886/143774706-d64a4e2a-d781-4d30-9dec-299fecac3c11.png)

**Comment by aborruso** [comment, marked as answer]
Vedi issue #24
